path_config.go

```diff
@@ -33,10 +33,11 @@ func pathConfig(b *kubeAuthBackend) *framework.Path {
 
 			"kubernetes_ca_cert": {
 				Type:        framework.TypeString,
-				Description: "PEM encoded CA cert for use by the TLS client used to talk with the API.",
+				Description: "Optional PEM encoded CA cert for use by the TLS client used to talk with the API.",
 				DisplayAttrs: &framework.DisplayAttributes{
 					Name: "Kubernetes CA Certificate",
 				},
+				Required: false,
```

Comment:
This is not immediately apparent, but this field does not have any [actual effect](https://github.com/hashicorp/vault/blob/5c4b49c457774b58fbb065dc7040237648f0e75c/sdk/framework/backend.go#L673) on the enforcement of whether a field schema is required or not. The `Required` bool here is purely for letting consumers of openapi (e.g. the UI) know whether the field should be treated as a required. 

Since the zero-value of this field is `false`, specifying this might not have any effect at all, even for openapi evaluation.

The removal of the check further below, is the right thing to do to remove the cert requirement though.

Reply:
The `Required` bool was added in just for future reference and documentation (lettings consumers know how the field is treated as mentioned)